library.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "esp32-camera",
+    "keywords": "esp32, camera",
+    "description": "esp32-camera",
+    "authors": [
+        {
+            "name": "Expressif"
+        }
+    ],
+
+    "build": {
+        "flags": "-Idriver/include -Idriver/private_include -Iconversions/include -Iconversions/private_include -Isensors/private_include"

```

Comment:
You can declare `flags` as an array for better visibility. Do we need `-fno-rtti`? See https://github.com/espressif/esp32-camera/blob/master/component.mk#L4

Reply:
I am unsure what this flag does.
I'm not from a c/c++ background, but believe this a flag passed to the c++ compiler?
Its easy enough to add if needed, but would rather know its purpose rather than just adding it blindly.